packages/git-credential-manager-core-deb/git-credential-manager-core-deb.pacscript

```diff
@@ -1,10 +1,10 @@
 name="git-credential-manager-core-deb"
 pkgname="git-credential-manager-core"
-gives="gcmcore"
+gives="gcm"
```

Comment:
Why are you changing this? Any specific reason?

Reply:
this is why https://github.blog/2022-04-07-git-credential-manager-authentication-for-everyone/ the name changed https://github.com/GitCredentialManager/git-credential-manager/releases/tag/v2.0.785

Reply:
Exactly what @cat-master21 mentioned. Additionally, they renamed the download artifact. That's why I changed gcmcore to gcm.

Reply:
I'm sorry, but I don't understand, if I have to do some more things. @Henryws wrote 

> Requested changes must be addressed to merge this pull request.

As I understand this, this is meant for gcmcore <-> gcm. In my opinion this is already addressed?!

Reply:
The requested change being the title change.

Reply:
@indika-dev.